core-bundle/src/Repository/TrustedDeviceRepository.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Repository;
+
+use Contao\BackendUser;
+use Contao\FrontendUser;
+use Doctrine\ORM\EntityRepository;
+
+class TrustedDeviceRepository extends EntityRepository
+{
+    public function findForBackendUser(BackendUser $user)
+    {
+        return $this->createQueryBuilder('td')
```

Comment:
You are currently having one column "member" and one column "user", but I prefer using one column "user_class/entity" and "user_id". This will allow one to add additional user entities, e.g. API users.

Reply:
Good point, it should actually be the username and entity, as that is what identifies a user in Symfony (not a user_id)

Reply:
But a username is subject to change...?